app/_src/gateway/admin-api/index.md

```diff
@@ -728,6 +728,235 @@ key_set_data: |
 
 ---
 
+<!-- vale off -->
+{% unless page.edition == "gateway" %}
+
+The API for configuring Kong Konnect Runtime Groups.
+
+
+
+
+{% assign prefix = "" %}
+{% if page.edition == "konnect" %}
+{% assign prefix = "/core-entities" %}
+{% endif %}
+
+
+| URL                | Description                                                                                                                         |
+| ---------                | -----------                                                                                                                         |
+| `https://us.api.konghq.com/v2/runtime-groups/{runtime_group_id}`                   | US Region Konnect Platform Base URL |
+| `https://eu.api.konghq.com/v2/runtime-groups/{runtime_group_id}` | EU Region Konnect Platform Base URL |
+       
+
+
+
+This API is similar to the [Kong Gateway admin API](/gateway/admin-api/) with a few major differences:
+
+* No `PATCH` methods.
+> `PATCH` methods are not yet available in the Konnect core entities endpoint. Update operations can be performed with the `PUT` method. 
+
+* Non-backwards compatible error responses.
+>  Error responses returned in Konnect core entities endpoint are not compatible with the error responses returned by the Kong Admin APIs.
+
+* `POST` methods do not support URL encoded forms.
+> The content types `application/x-www-form-urlencoded` and `multipart/form-data` are not supported in the Konnect core entities endpoint. Please use application/json.
+
+* All core entities endpoints include only the top level path.
+> For example, to retrieve a route, only `/routes/{route name or id}` is supported.
+
+{:.note}
+> Note: These gaps will be closed in future versions.
```

Comment:
Do we actually plan to close all these gaps? I think it might be ok to leave out this comment unless we're 100% sure.

Reply:
Will leave this out until we are more certain what will be updated. Will present the differences as facts for now.